web/src/invite.js

```diff
@@ -256,6 +256,27 @@ function open_invite_user_modal(e) {
 
     function invite_user_modal_post_render() {
         $("#invite-user-modal .dialog_submit_button").prop("disabled", true);
+        $("#invite-user-modal #email_invite_radio").prop("checked", true);
+
+        if (!page_params.is_admin) {
+            $("#generate_multiuse_invite_radio").prop("disabled", true);
+
+            $("#generate_multiuse_invite_radio")
+                .parent()
+                .parent()
+                .addClass("control-label-disabled");
+
+            $(".generate_multiuse_invite_radio_label").addClass("disabled_setting_tooltip");
+        } else {
+            $("#generate_multiuse_invite_radio").prop("disabled", false);
+
+            $("#generate_multiuse_invite_radio")
+                .parent()
+                .parent()
+                .removeClass("control-label-disabled");
+
+            $(".generate_multiuse_invite_radio_label").removeClass("disabled_setting_tooltip");
+        }
```

Comment:
Is there a special condition where this else block needs to be executed? It seems we don't need this else block because `#generate_multiuse_invite_radio` is already enabled when elements are rendered.